src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs

```diff
@@ -66,9 +66,10 @@ public static bool HasStarted(HttpContext context, out Activity? aspNetActivity)
         KeyValuePair<string, object?>[]? tags;
         if (context.Request?.Unvalidated?.Path is string path)
         {
-            tags = cachedTagsStorage ??= new KeyValuePair<string, object?>[1];
+            tags = cachedTagsStorage ??= new KeyValuePair<string, object?>[2];
 
             tags[0] = new KeyValuePair<string, object?>("url.path", path);
+            tags[1] = new KeyValuePair<string, object?>("http.request.method", path);
```

Comment:
I doubt that that the `path` should be passed as a value parameter here.

Reply:
You are correct... fixed in [f62e71c](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2023/commits/f62e71cb50a74305f654b363c3571a34cfd2afdc)